packages/indy-vdr/src/anoncreds/indyVdrAnoncredsRegistry.ts

```diff
@@ -392,26 +395,143 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
     revocationRegistryId: string,
     timestamp: number
   ): Promise<GetRevocationStatusListReturn> {
-    return {
-      resolutionMetadata: {
-        error: 'Not Implemented',
-        message: `Revocation list not yet implemented `,
-      },
-      revocationStatusListMetadata: {},
+    try {
+      const indySdkPoolService = agentContext.dependencyManager.resolve(IndyVdrPoolService)
+      const did = didFromRevocationRegistryDefinitionId(revocationRegistryId)
+
+      const pool = await indySdkPoolService.getPoolForDid(agentContext, did)
+
+      agentContext.config.logger.debug(
+        `Using ledger '${pool.indyNamespace}' to retrieve revocation registry deltas with revocation registry definition id '${revocationRegistryId}' until ${timestamp}`
+      )
+
+      const request = new GetRevocationRegistryDeltaRequest({
+        submitterDid: did,
+        revocationRegistryId,
+        toTs: timestamp,
+      })
+
+      agentContext.config.logger.trace(
+        `Submitting get revocation registry delta request for revocation registry '${revocationRegistryId}' to ledger`
+      )
+
+      const response = await pool.submitReadRequest(request)
+
+      agentContext.config.logger.debug(
+        `Got revocation registry deltas '${revocationRegistryId}' until timestamp ${timestamp} from ledger`
+      )
+
+      const { resolutionMetadata, revocationRegistryDefinitionMetadata } = await this.getRevocationRegistryDefinition(
+        agentContext,
+        revocationRegistryId
+      )
+
+      if (
+        !revocationRegistryDefinitionMetadata.issuanceType ||
+        typeof revocationRegistryDefinitionMetadata.issuanceType !== 'string'
+      ) {
+        return {
+          resolutionMetadata: {
+            error: `error resolving revocation registry definition with id ${revocationRegistryId}: ${resolutionMetadata.error} ${resolutionMetadata.message}`,
+          },
+          revocationStatusListMetadata: {
+            didIndyNamespace: pool.indyNamespace,
+          },
+        }
+      }
+
+      if (!response.result.data) {
+        return {
+          resolutionMetadata: {
+            error: 'notFound',
+            message: `Error retrieving revocation registry delta '${revocationRegistryId}' from ledger, potentially revocation interval ends before revocation registry creation`,
+          },
+          revocationStatusListMetadata: {},
+        }
+      }
+
+      return {
+        resolutionMetadata: {},
+        revocationStatusList: {
+          issuerId: response.result.identifier,
+          revRegId: response.result.revocRegDefId,
+          currentAccumulator: response.result.data.value.accum_to.value.accum,
+          revocationList: response.result.data?.value.revoked,
```

Comment:
As you can see in the `anonCredsRevocationStatusListFromIndySdk` method the transformation is a bit more involved than returned the `revoked` property: https://github.com/hyperledger/aries-framework-javascript/blob/main/packages/indy-sdk/src/anoncreds/utils/transform.ts#L58